### ISSUE TITLE
Change all storage doc comments to be regular comments

### ISSUE THREAD

**Issue by Braqzen** [FuelLabs/sway-applications#201]
### Motivation

CI is failing because of lots of breaking changes and updates being made.
This is an intermediate issue meant to unblock other PRs.

Change `///` to be `//` in the storage block for each app.

**Comment by Braqzen** [comment]
Closing in favor of #209
